apps/dashboard/app/(app)/apis/_components/api-list-card.tsx

```diff
@@ -50,7 +51,9 @@ export const ApiListCard = ({ api }: Props) => {
           <div className="flex items-center gap-2 min-w-0 max-w-[40%]">
             <Key className="text-accent-11 flex-shrink-0" />
             <div className="text-xs text-accent-9 truncate">
-              {keyCount > 0 ? `${keyCount} ${keyCount === 1 ? "Key" : "Keys"}` : "No data"}
```

Comment:
I'd say we just display it as "0 Keys", why would we say we don't have data? :D

Reply:
We have a similar setup at /ratelimits where we also show "No data". It's okay but If we don't like it we can get rid of it.

Reply:
It makes sense for rate limit cause we are showing a last used time.
But here we always have a number and having 0 keys is perfectly valid